user_guide/magento_entities.rst

```diff
@@ -88,52 +92,52 @@ Customer*):
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 From the *View* page of any shopping cart you can
 
-- Perform the actions specified in the Communication &  Collaboration section of the entity (*System → Entities → 
+- Perform the actions specified in the Communication &  Collaboration section of the entity (*System → Entities →
   Entity Management/Shopping Cart*):
 
-- Synchronize Data, i.e. upload the latest information for the cart/order from Magento and back (as defined by the 
+- Synchronize Data, i.e. upload the latest information for the cart/order from Magento and back (as defined by the
   synchronization settings).
 
-.. image:: ./img/magento_entities/view_carts.png
+  .. image:: ./img/magento_entities/view_carts.png
 
-.. important:: 
+.. important::
 
     Information for all the carts is updated once in a predefined period (default value is 5 minutes).
     However, it is strongly recommended to update a specific Cart record before you perform any actions with it.
 
 
-You can also place an order from the :ref:*View* page of every shopping cart with *Open* status (items in the cart have 
+You can also place an order from the *View* page of every shopping cart with *Open* status (items in the cart have
```

Comment:
@nnenasheva Actually, I wasn't able to figure out what should be referenced here. Can you help me?